AdobeStockAsset/Test/Integration/Model/GetAssetByIdTest.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\AdobeStockAsset\Test\Integration\Model;
+
+use Magento\AdobeStockAssetApi\Api\GetAssetByIdInterface;
+use Magento\AdobeStockAssetApi\Api\GetAssetListInterface;
+use Magento\Framework\Api\Search\DocumentInterface;
+use Magento\TestFramework\Helper\Bootstrap;
+use PHPUnit\Framework\TestCase;
+
+class GetAssetByIdTest extends TestCase
+{
+    /**
+     * @var GetAssetByIdInterface
+     */
+    private $getAssetById;
+
+    protected function setUp(): void
+    {
+        Bootstrap::getObjectManager()->configure([
+            'preferences' => [
+                GetAssetListInterface::class => GetAssetListMock::class
```

Comment:
Please reuse the existing client mock instead of `GetAssetListMock`

Reply:
Hi Sergii, okay I'll just wait for https://github.com/magento/adobe-stock-integration/pull/1803 to be merged. Thank you.

Reply:
@yolouiese it's merged now. Thanks!